### ISSUE TITLE
useCookie doesn't support `options` on remove

### ISSUE THREAD

**Issue by JoeDuncko** [streamich/react-use#1518]
**What is the current behavior?**

`useCookie` currently doesn't support `options` on `remove`. That means that you can't remove any scoped cookies. See https://stackoverflow.com/questions/54861709/cookies-removeabc-not-working-in-reactjs/55593030 and https://github.com/reactivestack/cookies/issues/16.

**Steps to reproduce it and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code and it doesn't have extra dependencies other than `react-use`. Paste the link to your [JSFiddle](https://jsfiddle.net) or [CodeSandbox](https://codesandbox.io) example below:**

```JavaScript
    const [cookie, setCookie, deleteCookie] = useCookie("cookie");

    setCookie(accessToken, {
            domain: document.domain.match(/[^.]*\.[^.]*$/)?.[0],
        });

    // This doesn't work because it doesn't pass the scope
    deleteCookie();

    // This is what needs to work, but since deleteCookie doesn't support `options`, it won't work
    deleteCookie({
            domain: document.domain.match(/[^.]*\.[^.]*$/)?.[0],
        });
```

**What is the expected behavior?**

**A little about versions:**  
- _OS_: macOS 10.15.6
- _Browser (vendor and version)_: Chrome 85.0.4183.102
- _React_: 16.13.1
- _`react-use`_: 15.3.4
- _Did this worked in the previous package version?_ No

**Comment by Boby** [comment]
for me I used `removeCookie('yourcookiename', { domain: '', path: '/' });` and it clears out but I had to refresh, anyway we can get the cookie domain?

**Comment by JoeDuncko** [comment]
> for me I used `removeCookie('yourcookiename', { domain: '', path: '/' });` and it clears out but I had to refresh, anyway we can get the cookie domain?

No, there is no way to get the cookie domain. It's a security function, as the dev should know the domain of the cookie they are manipulating.

**Comment by JoeDuncko** [comment]
Fixed in react-hookz/web in https://github.com/react-hookz/web/pull/117

**Comment by wildfrontend** [comment]
so , why not to merget this pull request ?

**Comment by JoeDuncko** [comment]
> so , why not to merge this pull request ?

Unfortunately, `react-use` is no longer supported, so this PR probably won't ever make it in. See [this issue](https://github.com/streamich/react-use/issues/1974) by the last active `react-use` maintainer. 

He's since moved moved on to [react-hookz/web](https://github.com/react-hookz/web).
